Micr.cs

```diff
@@ -28,19 +29,24 @@ protected enum FIELD
             ACCOUNT_NUMBER,
 
             /// <summary>
-            /// Field 5
+            /// Field 4
             /// </summary>
             ROUTING_NUMBER,
 
             /// <summary>
-            /// Field 6
+            /// Field 5
             /// </summary>
             EXTERNAL_PROCESSING_CODE,
 
+            /// <summary>
+            /// Field 6
+            /// </summary>
+            AUX_ON_US,
+
             /// <summary>
             /// Field 7
             /// </summary>
-            AUX_ON_US
+            ON_US
```

Comment:
This second AUX_ON_US should just be deleted instead of renamed.  ON_US is the ACCOUNT_NUMBER.